multihash-serialise/src/Codec/Serialise/Multihash.hs

```diff
@@ -27,12 +29,30 @@ import qualified Codec.Serialise as CBOR
 import qualified Codec.Serialise.Decoding as CBOR
 import qualified Codec.Serialise.Encoding as CBOR
 import           Control.Applicative (liftA2)
+import           Control.Monad ((>=>))
 import qualified Crypto.Hash as C
 import           Data.ByteArray (convert)
+import           Data.ByteString (ByteString)
+import qualified Data.ByteString.BaseN as BaseN
 import           Data.Multihash (Multihashable)
 import qualified Data.Multihash as Multi
 import           Data.Proxy (Proxy(..))
 
+-- | Decode a 'C.Digest' from a multihash- and base-n-encoded 'ByteString'.
+decodeAtBase
```

Comment:
I guess this doesn't _really_ belong here: either we're in the mülti wörld, or bareback bytestrings.

Reply:
Yeah, I feel you. However, this was the last little shim I had to write in `oscoin` to be able to tie the knot between these library and application code. Mind, I think the answer "this shim belongs into application code" is a perfectly reasonable answer, if we don't think this has enough general-value to be outsourced into library code 😉